index.html

```diff
@@ -393,7 +394,7 @@
                 <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.</p>
                 
-                <p>Initialize the <code>bubbles</code> attribute for the event to <code>true</code> if the event name is</p>
+                <p>Initialize the <code>bubbles</code> attribute and the <code>composed</code> [[!WHATWG-DOM]] attribute for the event to <code>true</code> if the event name is</p>
```

Comment:
The below list doesn't include `pointerenter` or `pointerleave` but you want those to be composed, right?  I suggest moving the text for composed up to the above sentence so that it applies to all pointer events.